src/Ng/SearchIndexFromCatalogCollector.cs

```diff
@@ -269,167 +255,6 @@ static float DetermineLanguageBoost(string id, string language)
-
-            string fullId = (package["namespace"] != null) ? string.Format("{0}.{1}", package["namespace"], package["id"]) : (string)package["id"];
-            Add(doc, "FullId", fullId, Field.Store.YES, Field.Index.ANALYZED, Field.TermVector.WITH_POSITIONS_OFFSETS);
-            Add(doc, "Namespace", (string)package["namespace"], Field.Store.YES, Field.Index.ANALYZED, Field.TermVector.WITH_POSITIONS_OFFSETS);
-
-            Add(doc, "TenantId", (string)package["tenantId"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.WITH_POSITIONS_OFFSETS);
-            Add(doc, "Visibility", (string)package["visibility"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.WITH_POSITIONS_OFFSETS);
-
-            doc.Add(new NumericField("PublishedDate", Field.Store.YES, true).SetIntValue(int.Parse(package["published"].ToObject<DateTime>().ToString("yyyyMMdd"))));
-
-            DateTime lastEdited = (DateTime)(package["lastEdited"] ?? package["published"]);
-            doc.Add(new NumericField("EditedDate", Field.Store.YES, true).SetIntValue(int.Parse(lastEdited.ToString("yyyyMMdd"))));
-
-            string displayName = String.IsNullOrEmpty((string)package["title"]) ? (string)package["id"] : (string)package["title"];
-            displayName = displayName.ToLower(CultureInfo.CurrentCulture);
-            Add(doc, "DisplayName", displayName, Field.Store.NO, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-
-            Add(doc, "Url", packageUrl.ToString(), Field.Store.YES, Field.Index.NO, Field.TermVector.NO);
-
-            Add(doc, "PackageContent", (string)package["packageContent"], Field.Store.YES, Field.Index.NO, Field.TermVector.NO);
-            Add(doc, "CatalogEntry", (string)package["@id"], Field.Store.YES, Field.Index.NO, Field.TermVector.NO);
-
-            Add(doc, "Listed", (string)package["listed"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-
-            //  The following fields are added for back compatibility with the V2 gallery
-
-            Add(doc, "OriginalVersion", (string)package["verbatimVersion"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-            Add(doc, "OriginalCreated", (string)package["created"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-            Add(doc, "OriginalLastEdited", (string)package["lastEdited"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-            Add(doc, "OriginalPublished", (string)package["published"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-
-            Add(doc, "LicenseNames", (string)package["licenseNames"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-            Add(doc, "LicenseReportUrl", (string)package["licenseReportUrl"], Field.Store.YES, Field.Index.NOT_ANALYZED, Field.TermVector.NO);
-
-            //TODO: add dependency summary
```

Comment:
We need the dependencies and supported frameworks for frameworks for V2 compatibility, correct?